File: Treinamento/C/Estrutura_sequencial/Questão04.c
#include <stdio.h>

int main() {
    double nota1, nota2, nota3, nota4;

    printf("Digite o primeira nota: ");
    scanf("%lf", &nota1);
    
    printf("Digite o segunda nota: ");
    scanf("%lf", &nota2);
    
    printf("Digite o terceira nota: ");
    scanf("%lf", &nota3);
    
    printf("Digite a quarta nota: ");
    scanf("%lf", &nota4);

    printf("Média: %lf\n", (nota1 + nota2 + nota3 + nota4) / 4);
}

File: Treinamento/Python/Exercícios/Estrutura_de_decisão/Questão04.py
print('*** Média escolar ***')

nota1 = float(input('Digite a primeira nota: '))
nota2 = float(input('Digite a segunda nota: '))

media = (nota1 + nota2) / 2 

print('Média: ' + str(media))

if media >= 7 and media <= 9.9:
    print('Parabéns, você foi aprovado')
elif media == 10:
    print('Parabéns, você foi aprovado com distinção')
else:
    print('Você foi reprovado')
    
